docs/UserGuide/9-Comparison/TSDB-Comparison.md

```diff
@@ -0,0 +1,400 @@
+
+  These two DBs are similar, while the first is based on HBase and the second is based on Cassandra.
+  Both of them provides RESTful style API.
+  
+  Interface: Restful API
+
+* TimeSacleDB - Time series database based on Relational Database
+
+  Interface: SQL
+
+Prometheus and Druid are also famous for time series data management. 
+However, Prometheus focuses on how to collect data, how to visualize data and how to alert warnings.
+Druid focuses on how to analyze data with OLAP workload. We omit them here.
+ 
+
+## Comparison 
+We compare the above time series database from two aspects: the feature comparison and the performance
+comparison.
+
+
+### Feature Comparison
+
+I list the basic features comparison of these databases. 
+
+Legend:
+- O: big support greatly
+- o: support
+- x: not support
+- :\-( : support but not very good
+- ?: unknown
+
+
+#### Basic Features
+
+| TSDB                        | IoTDB                       | InfluxDB   | OpenTSDB   | KairosDB   | TimescaleDB |   
+|-----------------------------|-----------------------------|------------|------------|------------|-------------|  
+| OpenSource                  | **o**                       | o          | o          | **o**      | o           |   
+| SQL\-like                   | o                           | o          | x          | x          | **O**       |   
+| Schema                      | "Tree\-based, tag\-based\"  | tag\-based | tag\-based | tag\-based | Relational  |   
+| Writing out\-of\-order data | o                           | o          | o          | o          | o           |   
+| Schema\-less                | o                           | o          | o          | o          | o           |   
+| Batch insertion             | o                           | o          | o          | o          | o           |   
+| Time range filter           | o                           | o          | o          | o          | o           |   
+| Order by time               | **O**                       | o          | x          | x          | o           |   
+| Value filter                | o                           | o          | x          | x          | o           |   
+| Downsampling                | **O**                       | o          | o          | o          | o           |   
+| Fill                        | **O**                       | o          | o          | x          | o           |   
+| LIMIT                       | o                           | o          | o          | o          | o           |   
+| SLIMIT                      | o                           | o          | x          | x          | ?           |   
+| Latest value                | O                           | o          | o          | x          | o           |
+
+**Details**
+
+* OpenSOurce:  
+ 
+  * IoTDB uses Apache License 2.0 and it is in Apache incubator. 
+  * InfluxDB uses MIT license. However, **the cluster version is not open sourced**.
```

Comment:
How about adding a separate section/item to highlight distributed functions?
This is an important advantage of iotdb over others.

Reply:
We can do that after our cluster mode is published.